onnx_chainer/functions/array.py

```diff
@@ -407,3 +407,34 @@ def convert_ResizeImages(func, opset_version, input_names, num_outputs,
         input_names.append(context.get_name(scales_param))
         return onnx_helper.make_node('Upsample', input_names, num_outputs,
                                      mode=mode),
+
+
+def convert_Stack(func, opset_version, input_names, num_outputs, context,
+                  parameters):
+    gb = onnx_helper.GraphBuilder()
+    axis = func.axis
+    if axis < 0:
+        axis = len(func.inputs[0].shape) + 1 + axis
+
+    # To use concat op, reshape every inputs add new axis
+    inputs = [gb.op('Unsqueeze', [name], axes=[axis]) for name in input_names]
+    gb.op('Concat', inputs, axis=axis)
+    return gb.nodes()
+
+
+def convert_Hstack(func, opset_version, input_names, num_outputs, context,
+                   parameters):
+    gb = onnx_helper.GraphBuilder()
+    axis = 0 if len(func.inputs[0].shape) == 1 else 1
```

Comment:
Even scalars can be hstacked? We need Unsqueeze for this case (I doubt this feature is useful, though...)?
```
>>> F.hstack([np.array(3), np.array(4)])
variable([3, 4])
```

Reply:
add test cases about input array is scalar, and support them both hstack and vstack